platform/viewer/src/connectedComponents/Viewer.js

```diff
@@ -295,7 +295,10 @@ class Viewer extends Component {
                 activeIndex={this.props.activeViewportIndex}
               />
             ) : (
-              <ConnectedStudyBrowser studies={this.state.thumbnails} />
+              <ConnectedStudyBrowser
+                studies={this.state.thumbnails}
+                studyMetadata={this.props.studies}
```

Comment:
Man, we really need to knock out a better way to do this

Reply:
Totally agree! In the next steps, we can probably add the study list in the redux store to avoid that.